src/aggregator/integration/integration_data.go

```diff
@@ -402,6 +422,8 @@ func computeExpectedAggregationBuckets(
 					values, err = addForwardedMetricToAggregation(values, mu.forwarded)
 				case timedMetric:
 					values, err = addTimedMetricToAggregation(values, mu.timed)
+				case passthroughMetric:
+					err = nil
```

Comment:
still pending?

Reply:
No.
Passthrough metrics need no aggregation, so the aggregation bucket calculation is skipped here as well.

Reply:
hm could you add a comment indicating the same thing